### PR TITLE
added a way to download data from openFDA and place it in the folder …

### PR DESCRIPTION
**Pull request by hl1**
read comments in get_data.py to understand what it is doing.

This will need several improvement later to 
1) differentiate between initial set up and update.
2) handling what needs to be done on the SQL side after my process.
3) restructuring our folders and files will lead to moving this file to another place
4) we should really use a `BASE_DIR `variable for all file and folder usage and manipulation. How to do it: https://stackoverflow.com/questions/3430372/how-do-i-get-the-full-path-of-the-current-files-directory
What django uses (see BASE_DIR) variable: https://github.com/django/django/blob/master/django/conf/project_template/project_name/settings.py-tpl

basically
`filepath = BASE_DIR / 'data' / 'myjsonfile.json'`
instead of:
`filepath = 'data/myjsonfile.json'`
5) we should have a settings file but that is a separate discussion in another PR.

### CODE REVIEW

**Comment by finish06** [comment]
Can you merge this with the master or would you like me to do it?  Let me know.  Thanks @hl1

**Comment by hl1** [comment]
@finish06 rebased.

**Comment by finish06** [comment]
I added the download script to the helpers folder.  Also, I made all paths absolute via `pathlib` library and renamed the final json file to match the rest of the code base.

**Comment by finish06** [comment]
@yevgenybulochnik - Please do the review on these code changes considering both @hl1 & myself have contributed.

File: app/helpers/get_data.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from urllib import request
+from datetime import datetime
+import zipfile
+from pathlib import Path
+
+
+def main():
+    # assign the name of the file that will be stored in the data/ directory.
+    # current output: data/2020-05-22_fda_data.zip 
+    data_filename = f"{Path.cwd() / 'data' / str(datetime.date(datetime.now()))}_fda_data.zip"
```
Comment (yevgenybulochnik) [inline app/helpers/get_data.py:10]: This line and subsequent lines fail if the `data` dir does not exist. We should either check to make sure this exists and warn the user it doesnt or have this directory created when this is run. In pathlib you can do this with the `mkdir` method  + flags `parents=True`, `exists_ok=True`

File: app/helpers/get_data.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+from urllib import request
+from datetime import datetime
+import zipfile
+from pathlib import Path
+
+
+def main():
```
[the remark this answers is not in the document]
Comment (yevgenybulochnik) [reply app/helpers/get_data.py:7]: scratch this comment I get why this was done

File: app/helpers/get_data.py (hunk 1 above)
Comment (finish06) [reply app/helpers/get_data.py:10]: I will merge and create an issue to be fixed.
